src/javascript/app/ng-wig/ngWigDirective.js

```diff
@@ -29,7 +29,10 @@ angular.module('ngWig')
               return;
             }
           }
-          scope.$emit('execCommand', {command: command, options: options});
+
+          if(!scope.editMode ) {
```

Comment:
it's not so good solution.
we should not put a condition, we should keep the event in a scope of exact editor